volume/drivers/coprhd/coprhd.go

```diff
@@ -72,54 +50,70 @@ func init() {
 }
 
 func Init(params volume.DriverParams) (volume.VolumeDriver, error) {
-	restUrl, ok := params["restUrl"]
+	host, ok := params["url"]
 	if !ok {
-		return nil, fmt.Errorf("rest api 'url' configuration parameter must be set")
+		return nil, fmt.Errorf("coprhd rest api 'url' configuration parameter must be set")
```

Comment:
if you are feeling extra ambitious, something in the common package would be nice like:

```
func NewRequiredError(field string) error {
  ...
}

func NewParameterMustBeSetError(parameter string) error {
  ...
}
```